VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -173,14 +167,17 @@ private void tryToUpdateParseForAppUpgrade(Context context){
         if (  previousVersion < curVersion ){
             ParseInstallation installation = ParseInstallation.getCurrentInstallation();
 
-            String language = Locale.getDefault().getLanguage();
-            installation.put("preferredLanguage",language);
+            Locale locale = Locale.getDefault();
+            installation.put("preferredLanguage",locale.getLanguage());
+            installation.put("preferredCountry",locale.getCountry());
```

Comment:
That sounds good to me. Realistically we're not properly dealing with the app language changing while it's running anyway.